app/api/utils.py

```diff
@@ -95,3 +97,47 @@ async def penalize(db, uid: UUID):
 
         if not res:
             raise HTTPException(500)
+
+def get_uuid(id: str):
+    try:
+        return UUID(id)
+    except ValueError:
+        return None
+
+def get_uuid_from_path(path: str):
+    uuid = get_uuid(path)
+    if uuid:
+        return uuid
+    return None
```

Comment:
Isn't this an unnecessary wrapper? Should only need `get_uuid`

Reply:
Jepp, should be removed 🔥